examples/rust/get_started/examples/04-udp-transport-responder.rs

```diff
@@ -14,11 +14,18 @@ async fn main(ctx: Context) -> Result<()> {
     let udp = node.create_udp_transport().await?;
 
     // Create a UDP listener and wait for incoming datagrams.
-    udp.listen("127.0.0.1:4000").await?;
+    let bind = udp
```

Comment:
I think tcp listening and udp bind has different semantics, so we should not reuse the word. I would even say that udp bind is closer to tcp accept, rather than listen

Reply:
You're right, that makes sense. Still, would there be a word that we could use as a default when no arguments/options are specified?

Reply:
We can implement conversion `&str` => `UDPBindArguments` to simplify the example. Why would we want to rename the function for that?